webapp/components/LoginForm/LoginForm.vue

```diff
@@ -20,15 +20,28 @@
           name="email"
           icon="envelope"
         />
-        <ds-input
-          v-model="form.password"
-          :disabled="pending"
-          :placeholder="$t('login.password')"
-          icon="lock"
-          icon-right="question-circle"
-          name="password"
-          type="password"
-        />
+        <div class="password-wrapper">
+          <ds-input
+            v-model="form.password"
+            :disabled="pending"
+            :placeholder="$t('login.password')"
+            icon="lock"
+            name="password"
+            class="password-field"
+            :type="showPassword ? 'text' : 'password'"
+          />
+          <a
+            class="click-wrapper"
+            @mousedown="
+              (event) => {
+                showPassword = !showPassword
+                event.preventDefault()
+              }
+            "
+          >
```

Comment:
better use `@click` and call a method like `toggleShowPassword`:

```
     toggleShowPassword() {
       this.showPassword = !this.showPassword
     },
```

Reply:
@Mogge Is there a way to prevent a change of focus when clicking the Icon without using the mouse down event? I think it would look better if the green highlighting remains when click the hide and show password. The user would not have to change contexts. I have not found a way to achieve this using the @click.

Reply:
I found a dirty trick to do it. First you need a `ref="password"` on the `ds-input`. Then the method called after click is:
```javascript
     toggleShowPassword(event) {
       this.showPassword = !this.showPassword
       event.preventDefault()
       this.$nextTick(() => {
         this.$refs.password.$el.children[1].children[1].focus()
       });
     },
```
We then also could test, if the focus is on the input field after clicking show password in the spec.